archive/migration-docs/backup-and-protect.sh

```
@@ -122,6 +121,10 @@ Schedule regular backups to prevent data loss.
 - 💾 Unlimited backup space
 - 🌐 Access from anywhere
 EOF
+- 🔒 Secure cloud storage
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)